File: Assets/Scripts/Reset Wall.js
#pragma strict
import UnityEngine.UI; 

var Player : GameObject;
var INITIALxCoord : int;
var INITIALyCoord : int;
var INITIALzCoord : int;

var livesText : Text;


 function OnCollisionEnter () { 
 Player.transform.position = new Vector3(INITIALxCoord,INITIALyCoord,INITIALzCoord);

 }

File: Assets/Scripts/ResetWall.cs
using UnityEngine;
using System.Collections;
using UnityEngine.UI; 

public class ResetWall : MonoBehaviour {
	public Vector3 INITIALcoord;
	public GameObject Player;
	public static int hit;

	void Start () {
		hit = 0;
	}

	void OnCollisionEnter () {
		hit = 1;
		Player.transform.position = INITIALcoord;
	}
		
}
